Rolled up updates

jhutchings1 opened the pull request:
👋🏻  @SharonManyara I went ahead and took all of your pull requests from the past few days and I retargeted them against this new branch called `rolled-up-updates` using this technique. This will let us review all of the changes together on staging before putting them out on the production site. You can do this too if you are working on something and accidentally send a PR to the default branch before it's ready. Here are some instructions for you :) https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request

SharonManyara committed:
Add files via upload

Lifeguard job application

SharonManyara committed:
Add files via upload

SharonManyara committed:
Add files via upload

SharonManyara committed:
Update index.md

SharonManyara committed:
Update mountainview-pool.md

jhutchings1 committed:
Merge pull request #87 from Klahanie/SharonManyara-patch-10

Add files via upload

jhutchings1 committed:
Merge pull request #88 from Klahanie/SharonManyara-patch-12

Add files via upload

jhutchings1 committed:
Merge pull request #89 from Klahanie/SharonManyara-patch-13

Add files via upload

jhutchings1 committed:
Merge pull request #91 from Klahanie/SharonManyara-patch-14

Update mountainview-pool.md

jhutchings1 committed:
Fixing syntax error

jhutchings1 committed:
Remove extra link title

jhutchings1 committed:
Merge pull request #90 from Klahanie/SharonManyara-patch-13-1

Update index.md

jhutchings1 commented:
Also, it looks like your changes for the how to pay your dues document didn't have the same filename, so I'll have to go and change the link on that.

jhutchings1 committed:
Update link for ways to pay

jhutchings1 committed:
Remove outdated version

jhutchings1 committed:
Update broken link from file deletion

jhutchings1 committed:
Fix broken link

Filename didn't have hyphens, it had spaces

jhutchings1 commented:
@SharonManyara Do you mind clicking around the staging site to see if everything looks like you expect it to? https://klahanie.github.io/klahanie

SharonManyara reviewed (changes requested):
@jhutchings1 
i just realized the application says 2020 instead of 2021. This is the updated file 
[Aquatics Lifeguard Employment Application 2021.pdf](https://github.com/Klahanie/klahanie/files/5852141/Aquatics.Lifeguard.Employment.Application.2021.pdf)

jhutchings1 commented:
@SharonManyara Do you want to switch to the rolled-up-updates branch and make the change by uploading the new file? Don't want to cut off a learning opportunity.

SharonManyara commented on _amenities/pools/index.md:
```diff
@@ -17,6 +17,7 @@ show-title: true
 * [Mountainview Pool Schedule]({{site.url}}/amenities/pools/mountainview-pool.html)
 * [Swim Lessons]({{site.url}}/amenities/pools/swim-lessons.html)
 * [Pool rules]({{site.url}}/amenities/pools/pool-rules.html)
+* [Lifeguard Employment Application]({{site.url}}/files/Aquatics Lifeguard Employment Application 2020.pdf)
```
[Aquatics Lifeguard Employment Application 2021.pdf](https://github.com/Klahanie/klahanie/files/5913117/Aquatics.Lifeguard.Employment.Application.2021.pdf)

SharonManyara replied:
updating new file

SharonManyara replied:
@sspicer1

SharonManyara committed:
Delete Aquatics Lifeguard Employment Application 2020.pdf

SharonManyara committed:
Create Lifeguard Employment Application

SharonManyara committed:
Delete Aquatics Lifeguard Employment Application 2020.pdf

SharonManyara committed:
Add files via upload

SharonManyara committed:
Update Lifeguard Employment Application

SharonManyara commented:
@jhutchings1 how do i correct exit code 1?

jhutchings1 commented on _amenities/pools/Lifeguard Employment Application:
```diff
@@ -0,0 +1 @@
+({{site.url}}/files/Aquatics Lifeguard Employment Application 2021.pdf)
```
This file shouldn't exist. Delete it?

jhutchings1 commented:
> @jhutchings1 how do i correct exit code 1?

This is your problem, the file you are linking to doesn't exist. Double check the name of the file and confirm that it's the same in both places. 
- ./_site/amenities/pools/index.html
  *  internally linking to /files/Aquatics Lifeguard Employment Application 2020.pdf, which does not exist (line 0)
     <a href="/files/Aquatics Lifeguard Employment Application 2020.pdf">Lifeguard Employment Application</a>

SharonManyara committed:
Add files via upload

SharonManyara committed:
Delete Employment-Application-r.2019-new.pdf

SharonManyara committed:
Update and rename Lifeguard Employment Application to Aquatics Lifeguard Employment Application 2021

re-doing the process to correct exit code 1. Is this the proper format to add a pdf file.

SharonManyara committed:
Update index.md

@jhutchings1 @sspicer1 
I think i corrected error code 1 by making all files had the same name. I'm not adding the link to the site.

sspicer1 reviewed (changes requested):
looks like there is a conflict line 13-17

SharonManyara committed:
Update mountainview-pool.md

SharonManyara reviewed (changes requested):
@sspicer1 i corrected line 13-17 (i hope). Please double check staging site (Thank you!)

jhutchings1 committed:
Remove redundant file and fix link

jhutchings1 committed:
Merge branch 'master' into rolled-up-updates

jhutchings1 committed:
Fix broken links

jhutchings1 commented:
I went ahead and cleared up the merge conflicts and fixed the broken links that were getting flagged in the tests. Merging.
